backend/WSS/migrations/0006_convert_participant_to_user_profile.py

```diff
@@ -3,13 +3,70 @@
 from django.conf import settings
```

Comment:
Please remove the `Generated by Django ...` comment since it's not auto-generated anymore.